main.py

```diff
@@ -18,6 +18,11 @@
         'path': 'http://environment.gov.sk.ca/firefiles/',
         'title': 'Saskatchewan Daily Wildfire Situation Map',
     },
+    {
+        'pdf': 'firestodate.pdf',
+        'path': 'http://environment.gov.sk.ca/firefiles/',
+        'title': 'Daily Wildfire Situation Map',
```

Comment:
Upon review... I know the title of the PDF is "Daily Wildfire Situation Map" but that's too similar to the above (line 19). But this map is really focused on "fires to date" locations. So why not call it that?

```suggestion
        'title': 'Fires to Date',
```